src/description.ts

```diff
@@ -1,21 +1,91 @@
-import { DEFAULT_LOCALE_DESCRIPTION } from './constants'
-import { Descriptions, LocalizedJSONSchema } from './types'
+import { JSONSchema6, JSONSchema6Definition } from 'json-schema'
+import { DEFAULT_DESCRIPTION_LOCALE } from './constants'
+import { Descriptions, LocalizedJSONSchema, LocalizedJSONSchemaDefinition } from './types'
 import { isNotEmpty } from './utils'
 
-export function getDescriptions (schema: LocalizedJSONSchema, localeDescription: string = DEFAULT_LOCALE_DESCRIPTION): Descriptions {
+export function getDescriptions (schema: LocalizedJSONSchema, descriptionLocale: string = DEFAULT_DESCRIPTION_LOCALE): Descriptions {
   const description = isNotEmpty(schema.description) ? schema.description : undefined
   const descriptions = schema.descriptions
 
   if (descriptions == null) {
-    return description != null ? { [localeDescription]: description } : {}
+    return description != null ? { [descriptionLocale]: description } : {}
   }
 
-  if (description != null && !Object.keys(descriptions).includes(localeDescription)) {
-    descriptions[localeDescription] = description
+  if (description != null && !Object.keys(descriptions).includes(descriptionLocale)) {
+    descriptions[descriptionLocale] = description
   }
   return descriptions
```

Comment:
Since we never need all the locales but only the current one, this function can probably be simplified:
```typescript
function getDescription (schema: LocalizedJSONSchema, locale: string = DEFAULT_LOCALE_DESCRIPTION)
  const description = isNotEmpty(schema.description) ? schema.description : undefined
  return descriptions[locale] ?? description
}
```